features/collect_logs.feature

```diff
@@ -15,9 +15,11 @@ Feature: Command behaviour when attached to an Ubuntu Pro subscription
         # So the -error suffix does not appear there.
         Then stdout matches regexp:
         """
+        apt-news.service.txt
         build.info
         cloud-id.txt
         cloud-init-journal.txt
+        esm-cache.service.txt
         jobs-status.json
         livepatch-status.txt-error
         pro-journal.txt
```

Comment:
I don't think this is necessary because the extra units should go into pro-journal.txt, I'll run this test to confirm

Reply:
oh nvm this is for the status output, not the logs